manifests/o/Obsidian/Obsidian/1.3.5/Obsidian.Obsidian.locale.en-US.yaml

```diff
@@ -0,0 +1,38 @@
+# Created with YamlCreate.ps1 v2.2.7 $debug=QUSU.LF.7-3-4.Unix
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Obsidian.Obsidian
+PackageVersion: 1.3.5
+PackageLocale: en-US
+Publisher: Obsidian
+PublisherUrl: https://github.com/obsidianmd
+PublisherSupportUrl: https://obsidian.md/community
+PrivacyUrl: https://obsidian.md/privacy
+Author: Obsidian
+PackageName: Obsidian
+PackageUrl: https://obsidian.md
+License: Proprietary
+LicenseUrl: https://obsidian.md/license
+Copyright: Copyright (c) 2023 Obsidian
+CopyrightUrl: https://obsidian.md/terms
```

Comment:
Note: As the EULA link is now down, I couldn't find a copyright link exactly. The terms of service linked here does mention some copyright notices though.